docs-conceptual/azps-11.6.0/authenticate-interactive.md

```diff
@@ -12,8 +12,8 @@ title: Sign in to Azure PowerShell interactively
 Interactive logins to Azure offer a more intuitive and flexible user experience. Interactive login
 with Azure PowerShell allows users to authenticate to Azure directly through the PowerShell
 interface, which is useful for ad-hoc management tasks and for environments that require manual
-sign-in, such as those with multi-factor authentication (MFA). This method simplifies access for
-script testing, learning, and on-the-fly management without needing to pre-configure service
+sign-in, such as those with multifactor authentication (MFA). This method simplifies access for
```

Comment:
Interactive logins to Azure offers a more intuitive and flexible user experience